src/index.js

```diff
@@ -142,7 +142,7 @@ async function publishBundle(bundle, handlers, outputDir, isLocal, apiToken) {
 
   /** @type {import("./upload").BundleInfo} */
   const bundleInfo = {
-    sha: shasum.digest("hex"),
+    sha: `${shasum.digest("hex")}.js`,
```

Comment:
should we change this to not be named `sha`, since it has an extension?

Reply:
I can change it. How would you like it to be named?
I am assuming this would require changes in both Netlify Dev and the manifest loader though?